templates/parachain/README.md

```diff
@@ -242,6 +242,127 @@ npx @acala-network/chopsticks@latest --chain-spec <path/to/chain_spec.json>
 support it at this moment. It can still be used to test a runtime in a full setup where it is started alongside a
 relay chain network (see [Parachain Template node](#parachain-template-node) setup).
 
+#### Deploy to Paseo TestNet & See Your First Block
```

Comment:
Should we have 2 `#` instead of 4, and 3 for the rest of sub steps? This one feels like a header title.